posthog/migrations/0217_migrate_tags_v2.py

```diff
@@ -0,0 +1,82 @@
+# Generated by Django 3.2.5 on 2022-03-01 23:41
```

Comment:
What's with the two migrations? Is this just because they are in diff django apps?

Reply:
It's a weird consequence/code smell of splitting out event_definitions and property_definitions into a seperate enterprise app, which requires them to be migrated separately.